### PR TITLE
Add sound

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#4 
#### What does this implement/fix? Explain your changes.
This pull request will add several sound effects to program such as:
+ `inhale`
+ `exhale`
+ `sustain`
+ `retain`

It requires `playsound` library which is used for playing sound asynchronous through program run.

### CODE REVIEW

File: nafas/functions.py [hunk 1]
```diff
@@ -225,9 +253,11 @@ def run(program_data):
         time.sleep(unit / 2)
         for index, item in enumerate(ratio):
             if item != 0:
+                item_name = STEP_MAP[index]
+                play_sound(get_sound_path(SOUND_MAP[item_name]))
```
Comment (sepandhaghighi) [inline nafas/functions.py:257]: I think we need a general error handler here (for `play_sound` function calling)

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: nafas/functions.py (hunk 1 above)
Comment (sadrasabouri) [reply nafas/functions.py]: You mean like [this](https://github.com/sepandhaghighi/nafas/pull/23/commits/e16653f8a414b9f6982a4d9f4b2c69b53a37ca66)?

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your effort 💯 
Please consider the following : 

1. unused import 'SOUND_ERROR_MESSAGE'
2. To solve codefactor issue, please move `try - exception` to `playsound` function and add a `debug=False` flag to this function to print error only in debug mode.

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your effort 💯 
Please consider the following : 

1. unused import 'SOUND_ERROR_MESSAGE'
2. To solve codefactor issue, please move `try - exception` to `playsound` function and add a `debug=False` flag to this function to print error only in debug mode.

**Comment by sadrasabouri** [comment]
I've tried many things but if we `join` the thread instantly after `start`ing it, It seems to work like blocking manner.

Anyway this is the last modified version which one of the test's output became awkward, what can we do if we want to have parallel run and don't have this issue simultaneously?

**Comment by sepandhaghighi** [comment]
> I've tried many things but if we `join` the thread instantly after `start`ing it, It seems to work like blocking manner.
> 
> Anyway this is the last modified version which one of the test's output became awkward, what can we do if we want to have parallel run and don't have this issue simultaneously?

@sadrasabouri Thanks for your effort 💯 
You are right, `play_sound` function should return `Thread` object to join main thread later (at the end of each loop in `run` function)

For test :
```python3
play_sound(1,True).join()
```

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
LGTM 💯

**Pull request merged**